### ISSUE TITLE
Number in tags (V2 editor)

### ISSUE THREAD

**Issue by Link2Twenty** [thepracticaldev/dev.to#1585]
**Describe the bug**
When I want to use the tag `a11y` I have to paste in the tag as numbers aren't allowed.

**To Reproduce**

1. Go to the v2 editor
2. Input any number into the tags field

**Expected behavior**
Numeric input should be valid

**Comment by wolfhoundjesse** [comment]
When you navigate to /t/accessibility it redirects to /t/a11y. That isn't the case on my machine. I only mention it because I thought maybe the tag was being rewritten, too. It's not.

Rather than changing the validation for tags, why don't we ensure that a11y is in the dropdown for the v2 editor?

**Comment by benhalpern** [comment]
Yeah, @Link2Twenty this makes sense.

`a11y` is now in the dropdown. Its absence was an oversight.
